### PR TITLE
Add doc tiddlers for reduce filter run prefix functionality

### PR DESCRIPTION
**Pull request by btheado**
Add separate documentation tiddlers for the filter run prefix functionality following the pattern set by the `:sort` and `:map` filter run prefixes.

The examples were taken from the `reduce` operator while modifying them to take advantage of the ability to inline the subfilter variables.

Expanded the width of the filter text box so longer filters will fit better.

### CODE REVIEW

File: editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid [hunk 1]
```diff
@@ -34,8 +34,6 @@ If a run has:
 
 <<.tip "Compare named filter run prefix `:filter` with [[filter Operator]] which applies a subfilter to every input title, removing the titles that return an empty result from the subfilter">>
 
-<<.tip "Compare named filter run prefix `:reduce` with [[reduce Operator]] which is used to used to flatten a list of items down to a single item by repeatedly applying a subfilter.">>
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid:37]: Why is this deleted?

**Review by Jermolene — COMMENTED** [review]
Thanks @btheado, much appreciated – just one question

File: editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid (hunk 1 above)
Comment (btheado) [reply editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid:37]: I moved this tip into the new "Reduce Filter Run Prefix" tiddler. Would you prefer it be in both places?

Comment (saqimtiaz) [reply editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid:37]: @btheado I think having it in both places is preferable.

**Comment by saqimtiaz** [comment]
Many thanks for this @btheado

**Comment by Jermolene** [comment]
Thanks @btheado

File: editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid (hunk 1 above)
Comment (btheado) [reply editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid]: Thanks for the feedback. I restored the tip to both places.

**Comment by Jermolene** [comment]
Thanks @btheado

**Comment by btheado** [comment]
I notice I accidentally made this PR against the master branch instead of tiddlywiki-com. Hope it doesn't cause any trouble.

**Comment by Jermolene** [comment]
> I notice I accidentally made this PR against the master branch instead of tiddlywiki-com. Hope it doesn't cause any trouble.

Oops I didn't notice either. It's not too serious at this point; the changes will get pushed to tiddlywiki-com when we release v5.2.1
